apps/site/pages/en/learn/test-runner/collecting-code-coverage.md

```diff
@@ -0,0 +1,303 @@
+
+```text displayName="Directory Structure"
+.
+├── main.test.js
+├── src
+│   ├── age.js
+│   └── name.js
+```
+
+```text displayName="Coverage Report"
+ℹ start of coverage report
+ℹ -------------------------------------------------------------
+ℹ file         | line % | branch % | funcs % | uncovered lines
+ℹ -------------------------------------------------------------
+ℹ main.test.js | 100.00 |   100.00 |  100.00 |
+ℹ src/age.js   |  45.45 |   100.00 |    0.00 | 3-5 7-9
+ℹ src/name.js  | 100.00 |   100.00 |  100.00 |
+ℹ -------------------------------------------------------------
+ℹ all files    |  88.68 |   100.00 |   75.00 |
+ℹ -------------------------------------------------------------
+ℹ end of coverage report
+```
+
+`src/age.js` has less-than-optimal coverage in the report above, but with the `--test-coverage-exclude` flag, it can be excluded from the report entirely.
+
+```bash
+node --experimental-test-coverage --test-coverage-exclude=src/age.js --test main.test.js
+```
+
+```text displayName="New coverage report"
+ℹ start of coverage report
+ℹ -------------------------------------------------------------
+ℹ file         | line % | branch % | funcs % | uncovered lines
+ℹ -------------------------------------------------------------
+ℹ main.test.js | 100.00 |   100.00 |  100.00 |
+ℹ src/name.js  | 100.00 |   100.00 |  100.00 |
+ℹ -------------------------------------------------------------
+ℹ all files    | 100.00 |   100.00 |  100.00 |
+ℹ -------------------------------------------------------------
+ℹ end of coverage report
+```
+
+Our test file is also included in this coverage report, but we only want JavaScript files in the `src/` directory. The `--test-coverage-include` flag can be used in this case.
+
+```bash
+node --experimental-test-coverage --test-coverage-include=src/*.js --test main.test.js
+```
+
+```text displayName="New coverage report"
+ℹ start of coverage report
+ℹ ------------------------------------------------------------
+ℹ file        | line % | branch % | funcs % | uncovered lines
+ℹ ------------------------------------------------------------
+ℹ src/age.js  |  45.45 |   100.00 |    0.00 | 3-5 7-9
+ℹ src/name.js | 100.00 |   100.00 |  100.00 |
+ℹ ------------------------------------------------------------
+ℹ all files   |  72.73 |   100.00 |   66.67 |
+ℹ ------------------------------------------------------------
+ℹ end of coverage report
+```
+
+## Thresholds
+
+By default, when all tests pass, Node.js exits with code `0`, which indicates a successful execution. However, the coverage report can be configured to exit with code `1` when coverage is failing.
+
+Node.js currently supports thresholds for all three of the coverages supported:
+
+- `--test-coverage-lines` for line coverage.
+- `--test-coverage-branches` for branch coverage.
+- `--test-coverage-functions` for function coverage.
```

Comment:
maybe add docs link for theses flag ??

Reply:
Done. note that the links are broken until the API documentation is updated with the release of v22.8.0